distributed/dashboard/scheduler.py

```diff
@@ -334,6 +365,7 @@ def get(self):
     (r"individual-plots.json", IndividualPlots),
     (r"metrics", PrometheusHandler),
     (r"health", HealthHandler),
+    (r"events", EventsHandler),
```

Comment:
Given the experimental nature of this, I would prefer for it to claim a less broad name.  Alternatively, is there some suffix that is common for websockets endpoints by any chance?

Reply:
That's reasonable. I'm not sure if there is a common suffix for websockets, generally they seem to be suffix-less.

How about something longer like `eventstream` or `eventsocket`?

Reply:
Yes, either of those sound good to me.  We could also elevate this route in the future to a more canonical name if it takes off.